docs/api-testing/apifctl-cicd-integration.md

```diff
@@ -0,0 +1,391 @@
+---
+id: apifctl-cicd-integration
+title: CI/CD Platform Integration with apifctl
+sidebar_label: CI/CD Integration with apifctl
+description: "Using Sauce Labs API Testing or CLI, you can seamlessly integrate continuous API testing into your CI/CD pipeline."
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Description
+
+Execute API tests and interact with Sauce Labs API Testing (either locally or in a pipeline) using our API Testing CLI tool, `apifctl`.
+
+## Usage
+
+`$ docker run quay.io/saucelabs/apifctl [COMMAND] [OPTIONS]`
+
+## What You'll Need
+* A [Docker](https://docs.docker.com/get-docker/) account.
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+* Your Sauce Labs [Username](https://app.saucelabs.com/user-settings) and [Access Key](https://app.saucelabs.com/user-settings).
+
+### Adding Webhooks
+To utilize most `apifctl` functionalities, you'll need to add a webhook to your API Testing Project by either generating one or retrieve one from your Dashboard.
```

Comment:
@kimsauce you can retrieve the Webhooks in the Webhooks section, the same section where you can create it. Once the web hook has been created it will be saved in the section and can be used in the future

Reply:
thank you for clarifying! I've rewritten this part a little bit -- will share shortly.